src/server/serviceProvider/transformers/Community/Reviews/getGenres.transform.js

```diff
@@ -0,0 +1,32 @@
+const GetGenresTransform = {
+  event() {
+    return 'getGenres';
+  },
+
+  requestTransform(event, {}, connection) {
+    return new Promise((resolve, reject) => {
+      this.callServiceClient('cached/standard/community', 'getGenres')
+      .then((response) => {
+        resolve(response);
+      })
+      .catch((err) => {
+        reject(err);
+      });
+    });
+  },
+
+  responseTransform(response, {}) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Change this pattern to not be empty. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS3799)